extensions/shadcn-ui/src/documentation/shadcn-ui-documentation.ts

```diff
@@ -0,0 +1,61 @@
+/*
+Documentation pulled manually from https://github.com/shadcn/ui/tree/main/apps/www/content/docs
+For now we upload this file manually and update the date here
```

Comment:
Im cool with having this manually, but would be better if there was an API for this! have you tried reaching out to shad?

Reply:
@shadcn ☝🏼 please check 😎

Reply:
for context:

I can read that data from `https://github.com/shadcn/ui/tree/main/apps/www/content/docs` and can even display the `mdx` content in a detail item (similar to what we do on the components list), BUT there are a few things missing:

- The order, nothing tell's me the order from what I found (and imo order is important here)
- I don't have way to check the section's, so it would be just a single list no sections (but sections might be important)

I would suggest going with this version first and if @shadcn is ok on changing the api somehow to get the two things above possible from the api, then we would push a new improved version.

I am not aware of the side effects so take it with a grain of salt:

- If on the api sections could be represented as folders that would resolve the sections
-About the order, not sure, probably best suggestions than the ones I am thinking 😅

Reply:
@peduarte let's leave this improvement for a next iteration, @shadcn is really busy 🔥 this days and we can plan this later at a better time for him.

Reply:
ok